### DISCUSSION TITLE
使用paddleocr 在某些有花纹背景的证件里面效果很差

### DISCUSSION THREAD

**Discussion by xxllp** [PaddlePaddle/PaddleOCR#12560] [Q&A]
如题，请问哪位大佬能给一些可靠的解决方案

**Comment by GreatV** [comment]
最好在自己的数据集上重新训练一下

**Reply by xxllp** [reply]
但是这种就包含检测和识别了吧，而且需要多少训练数据呢

**Reply by GreatV** [reply]
请参考：https://github.com/PaddlePaddle/PaddleOCR/blob/main/doc/doc_ch/FAQ.md#q%E7%AE%80%E5%8D%95%E7%9A%84%E5%AF%B9%E4%BA%8E%E7%B2%BE%E5%BA%A6%E8%A6%81%E6%B1%82%E4%B8%8D%E9%AB%98%E7%9A%84ocr%E4%BB%BB%E5%8A%A1%E6%95%B0%E6%8D%AE%E9%9B%86%E9%9C%80%E8%A6%81%E5%87%86%E5%A4%87%E5%A4%9A%E5%B0%91%E5%BC%A0%E5%91%A2

**Comment by xxllp** [comment]
主要是我们现在可供训练的数据也不是很多，看起来还是要生成一些数据了
